lib/grape/error_formatter/xml.rb

```diff
@@ -4,13 +4,14 @@ module Xml
       extend Base
 
       class << self
-        def call(message, backtrace, options = {}, env = nil)
+        def call(message, backtrace, options = {}, env = nil, exception = '')
```

Comment:
@dblock what do you think would be better here, receive the exception class name as a string or the whole exception?